packages/trpc/server/routers/viewer/teams/getMemberAvailability.handler.ts

```diff
@@ -34,7 +35,15 @@ export const getMemberAvailabilityHandler = async ({ ctx, input }: GetMemberAvai
       },
     },
   });
-  const member = members?.find((m) => m.userId === input.memberId);
+
+  const membersWithSelectedCalendars = members.map((m) => {
+    return {
+      ...m,
+      user: enrichWithSelectedCalendars(m.user),
+    };
+  });
+
```

Comment:
TODO: Move membership query to repository amd then enriching can happen in repository.